lib/tictactoe/game.rb

```diff
@@ -4,6 +4,11 @@ module Tictactoe
     
     class Game 
     end
-    class Board 
+    class Board
+
+        attr_accessor :size
+        def initialize()
+            @size = Array.new(9,"-")
```

Comment:
I feel like `@size` does not describe this data well. It looks like this is an internal representation of the board state.